mlir/lib/Dialect/Rock/Transforms/ConvToGemm.cpp

```diff
@@ -775,15 +797,15 @@ LogicalResult backwardData(Conv2DBwdDataOp op, PatternRewriter &b) {
     // Embed y/x into {y/x}dot and {y/x}tilda (Why the
     // particular embed coefficients is in a presentation somewhere)
     llvm::StringMap<uint32_t> embedDims = expandNamesInPlace(
-        filterNames, {{"y", {"ydot", "ytilda"}}, {"x", {"xdot", "xtilda"}}});
+        filterNames, {{"0", {"0dot", "0tilda"}}, {"1", {"1dot", "1tilda"}}});
```

Comment:
Need to also make a loop that generates the "Ndot" and "Ntilda" strings, and can then make the tilda and slice scalars into arrays and make this section into loops.